test/helpers/sandbox/__main__.py

```diff
@@ -0,0 +1,103 @@
+import logging
+import pandas as pd
+from app.main import Rocket
+
+from test.helpers.dummy_device_factory import DummyGPS, DummyBrakes, DummyParachute
+from gyro import SandboxGyro
+from accelerometer import SandboxAccelerometer
+from altimeter import SandboxAltimeter
+
+
+class SandboxDeviceFactory(object):
+    def __init__(self, simulation_data):
+
+        self.accelerometer = SandboxAccelerometer(self.parse_dataframe(simulation_data, 'acceleration'))
+        self.gyro = SandboxGyro(self.parse_dataframe(simulation_data,'rate (°/s)'))
+        self.altimeter = SandboxAltimeter(self.parse_dataframe(simulation_data,'Altitude'))
+        self.gps = DummyGPS()
+        self.parachute = DummyParachute()
+
+
+    def parse_dataframe(self,dataframe,data):
+        return pd.concat([dataframe.filter(items=['# Time (s)']), dataframe.filter(like=data)],axis=1)
+
+
+class Sandbox(object):
+    def __init__(self, source_file='test/helpers/sandbox/simulation.csv'):
+        self.simulation_data = pd.read_csv(source_file)
```

Comment:
we should call `pd.read` in each of the classes that use the pandas functions....right now when you read accelerometer it is not clear that you aredealing with a pandas dataframe from a file...it would be cleaner to jsut pass the file in and let the accelerometer etc handle wheter it wants to use pandas  (or something else for that use case)